cmd/inspect/inspect_interfaces.go

```diff
@@ -121,75 +109,13 @@ func inspectInterfacesFn(_ *cobra.Command, _ []string) error {
 		return nil
 	}
 
-	err = printContainerInterfaces(ctx, containers, interfacesFormat)
```

Comment:
most of these "removes" in inspect/inspect_itnerfaces moved to the clab list file